core/controller/src/main/scala/whisk/core/controller/Actions.scala

```diff
@@ -89,7 +89,7 @@ object WhiskActionsApi {
     def requiredProperties = WhiskServices.requiredProperties ++
         WhiskEntityStore.requiredProperties ++
         WhiskActivationStore.requiredProperties ++
-        WhiskConfig.actionSequenceMaxLimit
+        Map(WhiskConfig.actionSequenceLimit -> null)
```

Comment:
See my comment in another commit about checking the type of this value - if it's not of type int, we want to fail fast at startup.